/**:
  ros__parameters:
    simulation_active: true
    use_sim_time: true
    contact_stiffness: -1.0
    contact_damping: -1.0
    lateral_friction: 1.0
    spinning_friction: 0.1
    rolling_friction: 0.1
    restitution: 0.9
    order: 5
    cutoff: 10